### PR TITLE
added HareerSakhi's profile

### LINKED ISSUE #594: Hareer Sakhi
image of profile

(This is just part of my profile)

### PR DESCRIPTION
**Pull request by HareerSakhi**
Resolve #594

### CODE REVIEW

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thanks for collaborating and contributing :)

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for your contribution! 👍 🎉

**Review by deephunt3r — APPROVED** [review]
LGTM 👍🏻

**Comment by HareerSakhi** [comment]
Hi, I've just updated my image of my profile because I added some stats to my profile README.md. Please send any feedback if I need any. Thank you @deephunt3r @Panquesito7 and @kaiwalyakoparkar for your comments (:

**Review by kaiwalyakoparkar — APPROVED** [review]
Thanks for collaborating and contributing :)

File: data.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "name": "Hareer Sakhi",
+    "githubUsername": "HareerSakhi",
+    "imageUrl": "https://user-images.githubusercontent.com/70807684/126065573-860e71e9-0dce-47fd-bd99-3ad77c9d7c0c.png",
+    "issueNumber": 594
+  },
```
Comment (schmelto) [inline data.json:7]: New profiles should be added to the end of the readme, maybe you can open a new pr and adjust this

**Comment by schmelto** [comment]
New profiles should be added to the end of the readme, maybe you can open a new pr and adjust this

**Comment by AvidCoder101** [comment]
Hi, @schmelto I have made a new PR with my new username because I have changed my old one. I will put my new one at the end of the file. Thank you (:

**Pull request merged**